ProcessLib/HydroMechanics/HydroMechanicsFEM-impl.h

```diff
@@ -216,6 +214,13 @@ void HydroMechanicsLocalAssembler<ShapeFunctionDisplacement,
         auto& eps = _ip_data[ip].eps;
         auto const& sigma_eff = _ip_data[ip].sigma_eff;
 
+        auto const T_ref =
```

Comment:
Has somebody a better idea?
I could make T_ref a required property, but since it is not needed in most of the cases I don't think that makes sense.